plugins/Essentials/config.yml

```diff
@@ -715,7 +715,7 @@ newbies:
   # If so, use this format, replacing {DISPLAYNAME} with the player name.
   # If not, set to ''
   #announce-format: ''
-  announce-format: '&dWelcome {DISPLAYNAME}&d to the server!'
+  announce-format: '&7Welcome {DISPLAYNAME}&7 to the &6&lSpigot&e&lCraft&7!'
```

Comment:
I really like the generic greeting. Some people may use this as a template server (aka us who are too lazy on setting up a nice looking development server).